src/Components/Form.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+const Form = (props) => {
+  const { newBookFunc } = props;
+  // const [title, setTitle] = useState('');
+  // const [author, setAuthor] = useState('');
+  const [newBook, setNewBook] = useState({
+    title: '',
+    author: '',
+  });
+
+  const handleChange = (e) => {
+    setNewBook({
+      ...newBook,
+      [e.target.name]: e.target.value,
+    });
+
+    // if (e.target.name === 'title') {
+    //   setTitle(e.target.value);
+    // } else if (e.target.name === 'author') {
+    //   setAuthor(e.target.value);
+    // }
```

Comment:
It's good practice to remove all the commented code from your project. Kindly remove the commented code. 👍🏻